QuickFixers.Data/Models/Clients.cs

```diff
@@ -6,13 +6,14 @@
 
 namespace QuickFixers.Data.Models
 {
-   public class Clients
+   public class Clients: User, IUser
     {
         public int ClientID { get; set; }
-        public int UserID { get; set; }
         public string Name { get; set; }
         public string PhoneNumber { get; set; }
         public string Address { get; set; }
         public int ZipCode { get; set; }
+        public decimal PreferredDistance { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
```

Comment:
Is PreferredDistance supposed to be a part of Client?

Reply:
I don't think we use this for clients but we do for adding a new user in one sql call. MySQL forces values not optional. It's due to the Implementing the IUser interface. 

I can set a default 0 just in case (since we don't know what type a user until after logging in)